### PR TITLE
[KRYS-5336] Add FAQ view to staking view

### PR DESCRIPTION
**Pull request by MrQKyber**

### CODE REVIEW

File: EarnModule/EarnModule/View/SectionHeaderFAQView.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import Foundation
+import UIKit
+import DesignSystem
+import Utilities
+
+protocol SectionHeaderFAQViewDelegate: class {
+    func didChangeExpandStatus(status: Bool, section: Int)
+}
+
+class SectionHeaderFAQView: UITableViewHeaderFooterView {
```
Comment (tungnguyen20) [inline EarnModule/EarnModule/View/SectionHeaderFAQView.swift:17]: It's easier if using a single cell for each FAQ question. And implement expand/collapse inside the cell.

File: EarnModule/EarnModule/View/StakingFAQView.swift [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  }
+  
+  func reloadDataSource() {
+    guard input != nil, !fileName.isEmpty else {
+      dataSource = []
+      return
+    }
+    let decoder = PropertyListDecoder()
+      let bundle = Bundle(for: type(of: self))
+    if let url = bundle.url(forResource: fileName, withExtension: "plist"),
```
Comment (tungnguyen20) [inline EarnModule/EarnModule/View/StakingFAQView.swift:62]: Please re-indent

File: EarnModule/EarnModule/View/StakingFAQView.swift [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  var fileName: String {
+    guard let input = input else {
+      return ""
+    }
+
+    var name = ""
+    if input.platform == "ankr" && input.token == "matic" && input.chainID == 1 {
+      name = "Ankr-matic-eth"
+    } else if input.platform == "ankr" && input.token == "matic" && input.chainID == 137 {
+      name = "ankr-matic-matic"
```
Comment (tungnguyen20) [inline EarnModule/EarnModule/View/StakingFAQView.swift:41]: File names should have same format, ankr is capitalized or not?

File: EarnModule/EarnModule/View/StakingFAQView.swift [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    let cm = viewModel.dataSource[section]
+    view.updateTitle(text: cm.model.question)
+    view.section = section
+    view.isExpand = cm.isExpand
+    view.delegate = self
+    return view
+  }
+  
+  func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+    let cm = viewModel.dataSource[section]
```
Comment (tungnguyen20) [inline EarnModule/EarnModule/View/StakingFAQView.swift:164]: Same to this

File: EarnModule/EarnModule/View/SectionHeaderFAQView.swift (hunk 1 above)
Comment (MrQKyber) [reply EarnModule/EarnModule/View/SectionHeaderFAQView.swift:17]: it used for calculate height purpose

**Comment by tungnguyen20** [comment]
I think we still need to improve the expanding/collapsing animation. The html content should be beautify and some UI need to be improved. However, I approved to avoid blocking QA.

**Review by tungnguyen20 — APPROVED** [review]

**Pull request merged**